src/containers/BooksForm.js

```diff
@@ -1,25 +1,79 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
 
-const BooksForm = () => {
-  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
-  return (
-    <form action="">
-      <label htmlFor="title" id="title-form-label">
-        Title:
-        <input type="text" id="title" />
-      </label>
-
-      <label htmlFor="category" id="category-form-label">
-        Category:
-        <select id="category">
-          { categories.map(category => <option value={category} key={`${category}_option`}>{category}</option>)}
-        </select>
-      </label>
-      <button type="submit">Submit</button>
-    </form>
-  );
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: Math.floor(Math.random() * 100),
+      title: '',
+      category: 'Action',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    if (event.target.id === 'title') {
+      this.setState({
+        title: event.target.value,
+      });
+    } else if (event.target.id === 'category') {
+      this.setState({
+        category: event.target.value,
+      });
+    }
+  }
+
```

Comment:
this can be shortened to something like this 
```
handleChange(event) {
  const name = event.target.name;

  this.setState({
    title: event.target.value
    [name]: event.target.value
  })
}

```